lib/gcpspanner/chromium_histogram_enums_test.go

```diff
@@ -49,6 +49,23 @@ func insertSampleChromiumHistogramEnums(ctx context.Context, t *testing.T, c *Cl
 	return m
 }
 
+func insertGivenSampleChromiumHistogramEnums(
```

Comment:
Same thing with removing insertSampleChromiumHistogramEnums and using your method.
```suggestion
func insertTestChromiumHistogramEnums(
```